File: javascript/src/models/ordering.ts
// this file is @generated
/** Defines the ordering in a listing of results. */
export enum Ordering {
  Ascending = "ascending",
  Descending = "descending",
}

export namespace OrderingUtil {
  export function _fromJsonObject(object: any): Ordering {
    return object;
  }

  export function _toJsonObject(self: Ordering): any {
    return self;
  }
}

File: kotlin/lib/src/main/kotlin/models/Ordering.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import kotlinx.serialization.json.Json
import kotlinx.serialization.json.encodeToJsonElement
import kotlinx.serialization.json.jsonPrimitive

@Serializable
public enum class Ordering {
    @SerialName("ascending") ASCENDING,
    @SerialName("descending") DESCENDING;

    override fun toString(): String = Json.encodeToJsonElement(this).jsonPrimitive.content
}
